yapapi/runner/__init__.py

```diff
@@ -329,7 +324,7 @@ async def find_offers():
                             await proposal.respond(builder.props, builder.cons)
                             emit(Event.ProposalResponded(prop_id=proposal.id))
                         except Exception as ex:
-                            emit(Event.ProposalFailed(sub_id=subscription.id, reason=str(ex)))
+                            emit(Event.ProposalFailed(prop_id=proposal.id, reason=str(ex)))
```

Comment:
This error has been reported by `mypy` but only after I added a return type annotation to `def find_offers()`.